src/components/converter/Form.js

```diff
@@ -139,6 +141,17 @@ function useConvertInputs() {
 }
 
 function FormSection() {
+  const [selectedOption, setSelectedOption] = useState(0)
+  const [tokenBalance, setTokenBalance] = useState(bigNum(0))
+  const options = useMemo(() => ['ANT', 'DAI', 'ETH', 'USD'], [])
+  const selectedTokenBalance =
+    useTokenBalance(options[selectedOption]) || bigNum(0)
```

Comment:
`useTokenBalance()` [always returns an object](https://github.com/aragon/anj.aragon.org/blob/master/src/web3-contracts.js#L117), I think we might want to test for `.eq(-1)` instead.